packages/components/src/combobox-control/stories/index.story.tsx

```diff
@@ -111,3 +122,19 @@ WithCustomRenderItem.args = {
 		);
 	},
 };
+
+/**
+ * You can disable options in the list
+ * by setting the `disabled` property to true
+ * for individual items in the option object.
+ */
+export const WithDisabledOptions = Template.bind( {} );
+const optionsWithDisabledOptions = countryOptions.map( ( option, index ) => {
+	return { ...option, disabled: index % 3 === 0 };
```

Comment:
Could this also use a comment (that we're intentionally making the first and every 3rd item disabled)?